### ISSUE TITLE
docker: Error response from daemon: manifest for edgedb/edgedb:opt not found: manifest unknown: manifest unknown.

### ISSUE THREAD

**Issue by marcj** [edgedb/imdbench#51]
trying to run a few benchmarks locally, but can't get it to work:

```
$ make load-edgedb
docker network inspect webapp-bench>/dev/null 2>&1 \
		|| docker network create \
			--driver=bridge \
			--opt com.docker.network.bridge.name=br-webapp-bench \
			webapp-bench
docker volume inspect webapp-bench-edgedb >/dev/null 2>&1 \
		|| docker volume create webapp-bench-edgedb
docker stop webapp-bench-edgedb >/dev/null 2>&1 || :
docker run --rm -d --name webapp-bench-edgedb \
		-v webapp-bench-edgedb:/var/lib/edgedb/data \
		-e EDGEDB_SERVER_SECURITY=insecure_dev_mode \
		--network=webapp-bench \
		-p 15656:5656 \
		edgedb/edgedb:opt
Unable to find image 'edgedb/edgedb:opt' locally
docker: Error response from daemon: manifest for edgedb/edgedb:opt not found: manifest unknown: manifest unknown.
See 'docker run --help'.
make: *** [docker-edgedb] Error 125
```

**Comment by elprans** [comment]
This was fixed in `4f95aa769e90bfba`, are you running the latest version?

**Comment by marcj** [comment]
Good point, mine was from 24. march when initially heard of it. master fixed it, thanks!
